pkg/apis/core/types.go

```diff
@@ -3380,6 +3397,9 @@ type NodeStatus struct {
 	// List of volumes that are attached to the node.
 	// +optional
 	VolumesAttached []AttachedVolume
+	// Status of the config assigned to the node
```

Comment:
clarify this only applies if dynamic config feature is used and a ConfigSource is specified for the node?

Reply:
will do

Reply:
Technically it also applies when dynamic config is enabled, but you've set a nil configSource.